### PR TITLE
Trus 1480

### PR DESCRIPTION
**Pull request by joepowell96**

### CODE REVIEW

File: app/assets/stylesheets/partials/_inset_text.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.govuk-inset-text{
+    font-family:'nta',Arial,sans-serif;
+    -webkit-font-smoothing: antialiased;
```
Comment (joepowell96) [inline app/assets/stylesheets/partials/_inset_text.css]: Assets frontend has a `.indent` class available.

File: app/controllers/RemoveTrusteeController.scala [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                                              val controllerComponents: MessagesControllerComponents,
+                                              removeView: RemoveIndexView,
+                                              require: RequiredAnswerActionProvider
+                                 )(implicit ec: ExecutionContext) extends RemoveIndexController with I18nSupport {
+
+  override val messagesPrefix : String = "removeATrustee"
+
+  override val form: Form[Boolean] = formProvider.apply(messagesPrefix)
+
+  def actions(draftId : String, index: Int) =
```
Comment (joepowell96) [inline app/controllers/RemoveTrusteeController.scala]: Merge in master and define `override def page(index: Int`) and then call page(index) from requiredAnswer and content.
Remove the actions there now defined in the interface.

File: app/queries/RemoveTrusteeQuery.scala [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+package queries
+
+import play.api.libs.json.JsPath
+import sections.Trustees
+
+final case class RemoveTrusteeQuery(index : Int) extends Settable[Boolean] {
+
+  override def path: JsPath = JsPath \ Trustees \ index
```
Comment (joepowell96) [inline app/queries/RemoveTrusteeQuery.scala]: Spec is missing.

File: app/views/components/inset_text.scala.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(messageKey: String, id: String)(implicit messages: Messages)
+
+<div id="inset-text--@id" class="govuk-inset-text" style="font-family:'nta',Arial,sans-serif;-webkit-font-smoothing: antialiased;-moz-osx-font-smoothing: grayscale;font-weight: 400;font-size: 19px;line-height: 1.25;color: #0b0c0c;padding: 15px;margin-top: 20px;margin-bottom: 20px;clear: both;border-left: 10px solid #b1b4b6;">
```
Comment (joepowell96) [inline app/views/components/inset_text.scala.html]: Use the indent class as above.

File: conf/messages.en [hunk 5]
```diff
@@ -984,3 +989,8 @@ propertyOrLandAddressYesNo.error.required = Select yes if the property or land h
 
 propertyOrLandAnswers.title = Check property or land details
 propertyOrLandAnswers.heading = Check property or land details
+
+trusteeRemoveYesNo.title = trusteeRemoveYesNo
+trusteeRemoveYesNo.heading = trusteeRemoveYesNo
```
Comment (joepowell96) [inline conf/messages.en]: Remove

**Comment by adamconder** [comment]
@joepowell96 Can you add a descriptive title and description.

**Pull request merged**